package proto_ff;

import "yd_fieldoptions.proto";


message E_TaskrewardTaskrewardItemDesc
{
	optional int32 m_bind = 1[(yd_fieldoptions.field_cname) = "绑定"];
	optional int64 m_id = 2[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_val = 3[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_prof = 4[(yd_fieldoptions.field_cname) = "职业"];
}

message E_TaskrewardTaskrewardAttrDesc
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int64 m_val = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_TaskrewardTaskreward
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "奖励id"];
	optional int32 m_tasktype = 2[(yd_fieldoptions.field_cname) = "任务类型"];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 m_guildexp = 4[(yd_fieldoptions.field_cname) = "公会经验"];
	optional int32 m_guildpoint = 5[(yd_fieldoptions.field_cname) = "公会贡献"];
	optional int32 m_skillid = 6[(yd_fieldoptions.field_cname) = "技能ID"];
	repeated E_TaskrewardTaskrewardItemDesc m_item = 7[(yd_fieldoptions.field_cname) = "物品", (yd_fieldoptions.field_arysize) = 5];
	repeated E_TaskrewardTaskrewardAttrDesc m_attr = 8[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_TaskrewardTaskreward
{
	repeated E_TaskrewardTaskreward E_TaskrewardTaskreward_List = 1[(yd_fieldoptions.field_arysize)=200];
}

message E_TaskrewardTasktype
{
	optional int32 m_tasktype = 1[(yd_fieldoptions.field_cname) = "任务类型"];
}


message Sheet_TaskrewardTasktype
{
	repeated E_TaskrewardTasktype E_TaskrewardTasktype_List = 1[(yd_fieldoptions.field_arysize)=20];
}
